ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -79,7 +81,7 @@ const styles = StyleSheet.create({
  */
 const IdpLoginScreen = () => {
   const dispatch = useIODispatch();
-  const { navigate } = useIONavigation();
+  const { replace } = useIONavigation();
```

Comment:
What about adding a comment explaining **WHY** here it's necessary to use `replace` instead of `navigate`? What do you think?

Reply:
I agree!

Reply:
addressed [here](https://github.com/pagopa/io-app/pull/6565/commits/1c1a515d4fd2f7a6741c94a7dfef048e7505e280)!